pkg/iac-providers/output/types.go

```diff
@@ -44,8 +44,9 @@ type ResourceConfig struct {
 
 // ContainerNameAndImage container name and image
 type ContainerNameAndImage struct {
-	Name  string `json:"name"`
-	Image string `json:"image"`
+	Name            string          `json:"name"`
+	Image           string          `json:"image"`
+	Vulnerabilities []Vulnerability `json:"vulnerabilities"`
```

Comment:
should the struct's name be changed as it now contain additional field? eg: `ContainerDetails` ?